Bussiness layer

jakub-moravec opened the pull request.

slavicek2010 committed:
Add JSF dependencies
Create JSF pages - destination pages complete, listing complete
Create JSF Custom validators for destinations
Create ViewResources - complete
Mock services

slavicek2010 committed:
Update README

slavicek2010 committed:
GUI complete
TODO - connect to services, catching exceptions

jakub-moravec committed:
Delete reservation with pw

jakub-moravec committed:
delete reservation with password

jakub-moravec committed:
Merge remote-tracking branch 'origin/bussinessLayer' into bussinessLayer

# Conflicts:
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/service/ReservationService.java

jakub-moravec committed:
clean

klimesf reviewed (changes requested):
Používáte tam omnifaces, ale nemáte je přidaný v pom.xml, takže se to nezbuildí takhle

slavicek2010 committed:
TODO - SOFT DELETE, reservation delete with password

Merge branch 'master' of https://github.com/klimesf/a4m36jee-project into jsf

# Conflicts:
#	README.md
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/model/Flight.java
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/model/Reservation.java
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/service/DestinationService.java
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/service/FlightService.java
#	src/main/java/cz/cvut/fel/a4m36jee/airlines/service/ReservationService.java
#	src/main/webapp/WEB-INF/web.xml
#	src/test/java/cz/cvut/fel/a4m36jee/airlines/rest/ReservationResourceTest.java

jakub-moravec committed:
transactional

jakub-moravec committed:
soft delete

jakub-moravec committed:
soft delete

klimesf reviewed (changes requested):
U Controlleru můžete místo @RequestScoped a @Named použít jen @Model :) dělá to to samý

Entity by neměly být jako @ManagedBean, ale dá se dosáhnout úplně stejná funkcionalita, když dáte field do controlleru kterej bude @Named a @Produces. [Mrkněte sem](https://github.com/KarelCemus/a4b33si/blob/master/src/main/java/cz/cvut/fel/controller/MemberController.java#L45).

klimesf committed:
Merge branch 'master' into bussinessLayer

jakub-moravec committed:
Merge remote-tracking branch 'origin/master' into bussinessLayer
